numpy/lib/npyio.py

```diff
@@ -2225,6 +2225,9 @@ def ndfromtxt(fname, **kwargs):
     """
     Load ASCII data stored in a file and return it as a single array.
 
+    .. deprecated:: 1.17
+        np.ndfromtxt is deprecated
```

Comment:
Again, mention the replacement here, possibly with a few more words than in the `DeprecationWarning`.
```rst
`ndfromtxt` is a deprecated alias of `genfromtxt` which overwrites the ``usemask`` argument with `False` even when explicitly called as ``ndfromtxt(..., usemask=True)``. Use `genfromtxt` instead.
```